chessboard_detector:
  chess_rows: 12
  chess_cols: 9
  square_size: 0.05
  image_topic: "/vins_estimator/estimator/rgb_kf_image"
  pose_topic: "/vins_estimator/estimator/rgb_kf_quaternion"
  camera_frame: "camera_link"
  chessboard_frame: "chessboard"
